experimenter/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -1289,6 +1290,141 @@ def test_update_required_excluded_experiments_with_branches(self):
             branch_slug=excluded.reference_branch.slug,
         ).get()
 
+    @parameterized.expand(
+        [
+            (False, True, True),
+            (False, False, False),
+        ]
+    )
+    def test_can_update_subscribers(
+        self, subscriber_exists_initially, subscribed, subscriber_exists
```

Comment:
subscriber_exists_initially is always False

Reply:
additionally, subscribed == subscriber_exists for both cases

Reply:
👍